solidity/contracts/domains/noto/NotoFactory.sol

```diff
@@ -2,7 +2,6 @@
 pragma solidity ^0.8.20;
 
 import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
-import {Clones} from "@openzeppelin/contracts/proxy/Clones.sol";
 import {ERC1967Proxy} from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
```

Comment:
Was this just not being used before?

Reply:
I think we had just used Clones in some other places, and propagated the pattern to here... without connecting the dot that it then made the instances non-upgradeable (even though they implement UUPSUpgradeable, it doesn't work unless it's coupled with ERC1967).